### PR TITLE
Filter dependabot to only latest line and sample plugin

### PR DESCRIPTION
**Pull request by timja**
https://github.com/jenkinsci/bom/pull/265#issuecomment-654870763

Doesn't seem possible to include the root pom and not all submodules
So the root pom will be excluded with this, there's only one version in there which is the parent pom

Thoughts?

### CODE REVIEW

**Comment by jglick** [comment]
> So the root pom will be excluded with this, there's only one version in there which is the parent pom

Agreed, we can update that manually—a good tradeoff.

**Comment by oleg-nenashev** [comment]
> Doesn't seem possible to include the root pom and not all submodules
So the root pom will be excluded with this, there's only one version in there which is the parent pom

Was in my topics for Pligon POM 4.0 (changing Group ID as nuclear option). Have not really started

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
```
Comment (jglick) [inline .github/dependabot.yml:3]:
```suggestion
```

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "maven" # See documentation for possible values
+    directory: "/sample-plugin" # Location of package manifests
```
Comment (jglick) [inline .github/dependabot.yml:9]:
```suggestion
  - package-ecosystem: "maven"
    directory: "/sample-plugin"
```

File: .github/dependabot.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    reviewers:
+      - "jglick"
+    schedule:
+      interval: "daily"
+  - package-ecosystem: "maven" # See documentation for possible values
+    directory: "/bom-latest" # Location of package manifests
```
Comment (jglick) [inline .github/dependabot.yml:15]:
```suggestion
  - package-ecosystem: "maven"
    directory: "/bom-latest"
```

File: .github/dependabot.yml [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "maven" # See documentation for possible values
+    directory: "/sample-plugin" # Location of package manifests
+    reviewers:
+      - "jglick"
```
Comment (jglick) [inline .github/dependabot.yml:11]: Feel free to add yourself if you like!

**Comment by oleg-nenashev** [comment]
We could release Parent POM with 2 group IDs as a compatible solution, and gradually migrate (e.g. `io.jenkins.main.plugin-pom`)

**Comment by jglick** [comment]
@oleg-nenashev maybe are you talking about something else. The sole downside of this that I can see is that we would no longer be offered updates to: https://github.com/jenkinsci/bom/blob/0284682220a8231cbff4ae03163b3a49aada8983/pom.xml#L4-L7

File: .github/dependabot.yml (hunk 4 above)
Comment (timja) [reply .github/dependabot.yml:11]: I'm already subscribed 👅

**Comment by jetersen** [comment]
> @oleg-nenashev maybe are you talking about something else. The sole downside of this that I can see is that we would no longer be offered updates to:
> 
> https://github.com/jenkinsci/bom/blob/0284682220a8231cbff4ae03163b3a49aada8983/pom.xml#L4-L7

Maybe it can change the filter to include only the root pom?

**Comment by timja** [comment]
> BTW if you are bored:
> 
> https://github.com/jenkinsci/bom/blob/0284682220a8231cbff4ae03163b3a49aada8983/prep.sh#L43-L45

not that bored.

**Pull request merged**